api/middleware/authRequired.js

```diff
@@ -45,8 +45,8 @@ const authRequired = async (req, res, next) => {
       req.profile = profile;
     } else {
       // if profile doesn't already exist, create one
-      const profileObj = await makeProfileObj(verify.claims.sub);
-      const newProfile = await DB.create('profiles', profileObj);
+      const providerObj = await makeProfileObj(verify.claims.sub);
+      const newProfile = await DB.create('Providers', providerObj);
```

Comment:
Maybe change this from newProfile to newProvider, same on line 50 in the findById